lambda-buffers-codegen/data/rust-prelude-base.json

```diff
@@ -0,0 +1,73 @@
+{
+  "opaquesConfig": {
+    "Prelude.Map": [
+      "std",
+      "collections",
+      "BTreeMap"
+    ],
+    "Prelude.Set": [
+      "std",
+      "collections",
+      "BTreeSet"
+    ],
+    "Prelude.List": [
+      "std",
+      "vec",
+      "Vec"
+    ],
+    "Prelude.Either": [
+      "std",
+      "result",
+      "Result"
+    ],
+    "Prelude.Maybe": [
+      "std",
+      "option",
+      "Option"
+    ],
+    "Prelude.Bytes": [
+      "std",
+      "vec",
+      "Vec<u8>"
+    ],
+    "Prelude.Text": [
+      "std",
+      "string",
+      "String"
+    ],
+    "Prelude.Char": [
+      "char"
+    ],
+    "Prelude.Integer": [
+      "num_bigint",
+      "",
+      "BigInt"
+    ],
```

Comment:
Right, I had that in Purs, I had a Maybe field if I recall properly. https://github.com/mlabs-haskell/lambda-buffers/blob/617ac9202cd9e3bc67ff85bdc3d7e2fd6ec2d13a/lambda-buffers-codegen/src/LambdaBuffers/Codegen/Purescript/Syntax.hs#L30

Reply:
This is a bit different, as we have a package name, but no module.
A nicer way would be to allow a list of modules, eg:
```hs
data Qualified a
  = Qualified'Builtin a
  | Qualified'LibRef CrateName [ModuleName] a
  deriving stock (Eq, Ord, Show)
```

Reply:
I think I'll refactor this